pypsrp/wsman.py

```diff
@@ -19,13 +19,17 @@
 from pypsrp.negotiate import HTTPNegotiateAuth
 from pypsrp._utils import to_string, to_unicode, get_hostname
 
-HAS_CREDSSP = True
-CREDSSP_IMP_ERR = None
 try:
     from requests_credssp import HttpCredSSPAuth
 except ImportError as err:  # pragma: no cover
-    HAS_CREDSSP = False
-    CREDSSP_IMP_ERR = err
```

Comment:
keeping this exception around is bad news for the GC in py3 because it has the .__traceback__ attached

Reply:
Would you be able to explain this any more? I use this in a few different places but if it's bad I would like to understand why.

Reply:
>  Exceptions are cleared because with the traceback attached to them, they form a reference cycle with the stack frame, keeping all locals in that frame alive until the next garbage collection occurs.

https://docs.python.org/3/reference/compound_stmts.html#the-try-statement

And it's not needed here, because you only need the str

Reply:
@jborean93 the reason it's bad is say someone did:


```python
def foo():
    x = b"\x00" * (2**32)
    import pypsrp
```
then that huge byte object would never be garbage collected as there would always be a global reference to it in:

`sys.modules["pypsrp"].CREDSSP_IMP_ERR.__traceback__.tb_frame.f_back.f_locals["x"]`

Reply:
Thank you very much for the explanation, it makes total sense and was not something I had considered before.